composer.json

```diff
@@ -1,14 +1,42 @@
 {
-    "name": "drupal-composer/drupal-project",
-    "description": "Project template for Drupal 8 projects with composer",
-    "type": "project",
-    "license": "GPL-2.0+",
     "authors": [
         {
             "name": "",
             "role": ""
         }
     ],
+    "autoload": {
```

Comment:
I like the original ordering better. This is just alphabetical but it is of little use. I think this better remain with contextual ordering.